### PR TITLE
Add handling for Cheng-Prusoff values in mtenn.PIC50Readout class

### PR DESCRIPTION
**Pull request by kaminow**
As of mtenn v0.3.0, the `PIC50Readout` class takes values for the Cheng-Prusoff equation. Update our code to interface properly with that.

### CODE REVIEW

File: asapdiscovery-ml/asapdiscovery/ml/utils.py [hunk 1]
```diff
@@ -250,6 +258,8 @@ def build_model(
             config = {}
     elif (type(config) is str) or isinstance(config, Path):
         config = parse_config(config)
+    elif type(config) is not dict:
```
Comment (hmacdope) [inline asapdiscovery-ml/asapdiscovery/ml/utils.py:261]: Long term may be an idea to move these configs to .... you guessed it, schema. But no need in this PR.

File: asapdiscovery-ml/asapdiscovery/ml/utils.py [hunk 2]
```diff
@@ -1722,9 +1745,15 @@ def update_loss_dict(split, compound_id, target, in_range, uncertainty, pred, lo
 
         # Stop if loss has gone to infinity or is NaN
         if (
-            np.isnan(epoch_val_loss)
+            np.isnan(epoch_train_loss)
```
Comment (hmacdope) [inline asapdiscovery-ml/asapdiscovery/ml/utils.py:1748]: Move to a function called `check_valid` or something like that this is a few too many `or`.

**Review by hmacdope — CHANGES_REQUESTED** [review]
Looking good, see comments. :)

File: asapdiscovery-ml/asapdiscovery/ml/utils.py (hunk 1 above)
Comment (kaminow) [reply asapdiscovery-ml/asapdiscovery/ml/utils.py:261]: yep that's a good idea. opened #450 to track this

**Review by hmacdope — APPROVED** [review]
Looks good to me @kaminow :)

**Pull request merged**